run_dir/design/running_notes_tab.html

```diff
@@ -2,33 +2,33 @@
     <div class="tab-pane fade" id="tab_running_notes_content">
         <!-- add running note -->
         <form action="" method="POST" id="running_notes_form" role="form">
-            <div class="panel panel-info">
-                <div class="panel-heading">Add New Running Note</div>
-                <div class="panel-body">
+            <div class="card text-dark info-border mb-3 mt-3">
+                <div class="card-header info-bg">Add New Running Note</div>
+                <div class="card-body">
                     <div class="row">
                         <div class="col-md-7 form-inline">
                           <label>Choose category:</label>
-                          <div class="btn-toolbar rn-categ" data-toggle="buttons" style="margin-top:5px">
-                             <button class="btn btn-xs btn-info" value="Decision" data-toggle="tooltip" title="For when an executive decision has been made">Decision <span class="fa fa-thumbs-up"</span></button>
-                             <button class="btn btn-xs btn-success" value="Lab" data-toggle="tooltip" title="For lab-related work">Lab <span class="fa fa-flask"</span></button>
-                             <button class="btn btn-xs btn-warning" value="Bioinformatics" data-toggle="tooltip" title="For all bioinformatics work">Bioinformatics <span class="fa fa-laptop-code"</span></button>
-                             <button class="btn btn-xs btn-usr" value="User Communication" data-toggle="tooltip" title="For notes influenced by user-contact">User Communication <span class="fa fa-people-arrows"</span></button>
-                             <button class="btn btn-xs btn-danger" value="Administration" data-toggle="tooltip" title="For notes involving documentation">Administration <span class="fa fa-folder-open"</span></button>
-                             <button class="btn btn-xs btn-imp" value="Important" data-toggle="tooltip" title="For when a note needs to be highlighted">Important <span class="fa fa-exclamation-circle"</span></button>
-                             <button class="btn btn-xs btn-devi" value="Deviation" data-toggle="tooltip" title="For notes about a deviation">Deviation <span class="fa fa-frown"</span></button>
+                          <div class="btn-toolbar rn-categ mt-2" data-toggle="buttons">
+                             <button class="btn btn-sm btn-info mr-2 text-light" value="Decision" data-toggle="tooltip" title="For when an executive decision has been made">Decision <span class="fa fa-thumbs-up"></span></button>
```

Comment:
I think the "Decision" and "Bioinformatics" buttons are now a bit too bright?

Reply:
Thats the default colour scheme on bootstrap 5

Reply:
Done!